### PR TITLE
SUS-2257 → migrate Wall history to IPv6

### PR DESCRIPTION
**Pull request by ghost**
https://wikia-inc.atlassian.net/browse/SUS-2257

1. started with a little clean up
2. added `post_user_ip_bin` for storing IPv6 binary data (the current data needs to be migrated from `post_user_ip` before it is dropped)
3. changed `WallHistory` to use the new column and proper data conversion

@macbre @TK-999 

Don't merge it yet. All wikis created before releasing this change have to be migrated. All wikis created after releasing this change will have the right schema automatically.

### CODE REVIEW

File: extensions/wikia/Wall/WallHistory.class.php [hunk 1]
```diff
@@ -156,7 +156,8 @@ private function internalAdd( $parentPageId, $postUserId, $postUserName, $isRepl
 				'parent_page_id' => $parentPageId,
 				'post_user_id' => $postUserId,
 				'post_ns' => MWNamespace::getSubject( $ns ),
-				'post_user_ip' => ( intval( $postUserId ) === 0 ? $this->ip2long( $postUserName ) : null ),
+				'post_user_ip' => null, // TODO SUS-2257 Mix <mix@fandom.com> Remove after migration.
```
Comment (WikiaTech) [inline extensions/wikia/Wall/WallHistory.class.php]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: extensions/wikia/Wall/WallHistory.class.php (hunk 1 above)
Comment (WikiaTech) [inline extensions/wikia/Wall/WallHistory.class.php]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: extensions/wikia/Wall/WallHistory.class.php (hunk 1 above)
Comment (WikiaTech) [inline extensions/wikia/Wall/WallHistory.class.php:159]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Review by macbre — APPROVED** [review]

**Review by mszabo-wikia — APPROVED** [review]

**Comment by macbre** [comment]
Can we merge this one?

**Comment by ghost** [comment]
I will merge it when I am done with databases.

**Pull request merged**